lib/web_ui/lib/src/engine/skwasm/skwasm_impl/renderer.dart

```diff
@@ -472,6 +472,19 @@ class SkwasmRenderer implements Renderer {
       surface.handle,
     ));
   }
+
+  @override
+  FutureOr<ui.Image> createImageFromTextureSource(Object textureSource, { required int width, required int height, required bool transferOwnership }) async {
+    if(!transferOwnership) {
```

Comment:
A space between the `if` and the conditional for consistency with our style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md

```suggestion
    if (!transferOwnership) {
```